inc/pkthlp.h

```diff
@@ -308,6 +309,129 @@ PktBuildTcpFrame(
     return TRUE;
 }
 
+//
+// Build a QUIC frame from the given parameters and paylaod.
```

Comment:
Like I said below, let's not use "frame" as none of this is specific to QUIC frames.

Reply:
To be clear, "frame" in this context refers to an Ethernet frame, not any QUIC frames within the Ethernet frame. The UDP and TCP helpers refer to frame with the same meaning. Unfortunately, there is no technically correct and non-overloaded term we can use, so let's be consistent and use "frame" here.

Reply:
That's actually a great case of confusion. This function creates a Quic packet, not an ethernet frame.
The output of this should be passed to "PktBuildUdpFrame" or "PktBuildTcpFrame" to encapsulate it further like mentioned in the comment.

I'll rename the function / parameters to make it clear.

Reply:
Ah, OK, if this is not a complete Ethernet frame, then I think the best term would be QUIC "datagram" unless the QUIC RFC has a more standardized term for its UDP payload.

Reply:
From the Quic RFC 8999: "QUIC endpoints exchange UDP datagrams that contain one or more QUIC packets."
"Quic packets" is the standardized term.

"Frame <-> Packet <-> Datagram <-> Packet <-> Frame", perfectly symmetric as all should be